reference/7/Microsoft.PowerShell.Core/About/about_Pipeline_Chain_Operators.md

```diff
@@ -0,0 +1,294 @@
+---
+ms.date:  09/30/2019
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell,cmdlet
+title:  about_Pipeline_Chain_Operators
+---
+
+# About Pipeline Chain Operators
+
+## Short description
+
+Describes chaining pipelines with the `&&` and `||` operators in PowerShell.
+
+## Long description
+
+> [!NOTE]
+> This is an experimental feature. For more information see
+> [about_Experimental_Features](about_Experimental_Features.md).
+
+Beginning in PowerShell 7, PowerShell implements the `&&` and `||` operators to
+conditionally chain pipelines. These operators are known in PowerShell as
+*pipeline chain operators*, and are similar to
+[AND-OR lists](https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_09_03)
+in POSIX shells like bash, zsh and sh, as well as
+[conditional processing symbols](/previous-versions/windows/it-pro/windows-xp/bb490954(v=technet.10)#using-multiple-commands-and-conditional-processing-symbols)
+in the Windows Command Shell (cmd.exe).
+
+The `&&` operator executes the right-hand pipeline, if the left-hand pipeline
+succeeded. Conversely, the `||` operator executes the right-hand pipeline if
+the left-hand pipeline failed.
+
+These operators use the `$?` and `$LASTEXITCODE` variables to determine if a
+pipeline failed. This allows you to use them with native command and not just
+with cmdlets or functions. For example, this could be used for conditional
```

Comment:
I want to stay away from the specifics of the example this early in the doc. I'll give a better example below

Reply:
See my updates